dev-docs/source/AlgorithmProfiler.rst

```diff
@@ -0,0 +1,40 @@
+==============================
+Work flows algorithm profiling
+==============================
+
+.. contents:: Contents
+    :local:
+
+Summary
+^^^^^^^
+
+Due to the need of investigation of algorithms performance issues, the proper method
+is introduced. It consists two to parts: special mantid build and analytical tool.
+Available for Linux only.
+
+Mantid build
+^^^^^^^^^^^^
+
+To build mantid for profiling run ``cmake`` with the additional option ``-DPROFILE_ALGORITHM_LINUX=ON``.
```

Comment:
`-DALGORITHM_PROFILER=ON`?

Reply:
"To build `Mantid` for..." ?

Reply:
Maybe the other flag would be needed for another os because of different profile functionality, so I'd leave LINUX in the flag. If somebody would implement something for other systems then would rename flag.